crates/pathfinder/src/state/block_hash.rs

```diff
@@ -163,7 +163,8 @@ mod meta {
     pub fn for_chain(chain: Chain) -> &'static BlockHashMetaInfo {
         match chain {
             Chain::Mainnet => &MAINNET_METAINFO,
-            Chain::Goerli => &TESTNET_METAINFO,
+            Chain::Testnet => &TESTNET_METAINFO,
+            Chain::Integration => todo!("Block hash meta data TBD"),
```

Comment:
I'm not entirely sure how to determine this. Currently it needs to be the beginning of the 0.9.1 blocks.. one way is to just let it run until it no longer works?

@kkovaacs how did we (you) determine this last time?

Reply:
Mostly by trial-and-error, and running the `verify_block_hashes` tool to test...

Reply:
Unfortunately having `todo!()` here makes it impossible to sync since it leads to panic. I'd suggest adding metadata that effectively disables block hash verification -- and we can move forward with this later if we want.

```suggestion
            Chain::Integration => &INTEGRATION_METAINFO,
```

and the metainfo is:

```rust
    const INTEGRATION_METAINFO: BlockHashMetaInfo = BlockHashMetaInfo {
        first_0_7_block: StarknetBlockNumber::new_or_panic(0),
        not_verifiable_range: Some(
            StarknetBlockNumber::new_or_panic(0)..StarknetBlockNumber::new_or_panic(1000000),
        ),
        fallback_sequencer_address: SequencerAddress(starkhash!(
            "046a89ae102987331d369645031b49c27738ed096f2789c24449966da4c6de6b"
        )),
    };
```

Reply:
Thanks!